doc/hardware.rst

```diff
@@ -0,0 +1,426 @@
+Hardware
+--------
+
+Identifying your serial interface
```

Comment:
Identifying your ~~serial~~ interface

Reply:
Removed serial and fixed a grammar issue in the sentence about the number of pins in a parallel interfaced device.